### PR TITLE
Bug 1017966 - Disable call button when dialer field is empty

### PR DESCRIPTION
**Pull request by rohan013**
The call button will be disabled if the dialer field is empty

### CODE REVIEW

**Comment by try-server-hook** [comment]
[rohan013 Rohan Rawat (rohan013)](https://github.com/rohan013) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e05af60f3aa0)

File: apps/communications/dialer/test/marionette/keypad_test.js [hunk 1]
```diff
@@ -114,4 +114,16 @@ marionette('Dialer > Keypad', function() {
     });
     assert.ok(true, 'cleaned the phone number view');
   });
+
+  test('Enabling and disabling the dialer call button', function() {
+    var callButton;
+
+    callButton = subject.client.findElement(selectors.keypadCallBarCallAction);
+    assert.ok(true, callButton.hasAttribute('disabled'));
```
Comment (DouglasSherk) [inline apps/communications/dialer/test/marionette/keypad_test.js]: ```assert.isTrue(callButton.hasAttribute('disabled'));``` .

File: apps/communications/dialer/test/marionette/keypad_test.js [hunk 2]
```diff
@@ -114,4 +114,16 @@ marionette('Dialer > Keypad', function() {
...
+
+    typeNumber();
+    callButton = subject.client.findElement(selectors.keypadCallBarCallAction);
+    assert.ok(true, !callButton.hasAttribute('disabled'));
```
Comment (DouglasSherk) [inline apps/communications/dialer/test/marionette/keypad_test.js]:
```js
assert.isFalse(callButton.hasAttribute('disabled'));
```

File: apps/communications/dialer/test/marionette/keypad_test.js [hunk 3]
```diff
@@ -114,4 +114,16 @@ marionette('Dialer > Keypad', function() {
     });
     assert.ok(true, 'cleaned the phone number view');
   });
+
+  test('Enabling and disabling the dialer call button', function() {
```
Comment (DouglasSherk) [inline apps/communications/dialer/test/marionette/keypad_test.js]: These changes can go in ```apps/communications/dialer/test/unit/keypad_test.js``` instead. The structure will be a bit different, though.

**Comment by try-server-hook** [comment]
[rohan013 Rohan Rawat (rohan013)](https://github.com/rohan013) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=eb0c5f237e75)

**Comment by try-server-hook** [comment]
[rohan013 Rohan Rawat (rohan013)](https://github.com/rohan013) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=21066417e904)

**Pull request merged**